Add initial objectives (#3)
Code-the-Dream-School/react-curriculum-v4: learns-app-content/week-01/objectives.md

<!-- h1, h2 already used by CTD Learns -->
### Topics Covered

- Intro to React
- App installation
- Project walk-through

### Lesson Objectives

By the end of this lesson, we will:

#### Objective 1: Intro to React

- Understand React at a high level

> - Explain the advantages React for developing web applications

Reviewer: ```suggestion
- Highlight the key advantages of React in web development.
```
